### ISSUE TITLE
Does not work in mac

### ISSUE THREAD

**Issue by kishaningithub** [jamesandres/ColumnCopy#21]
Can you kindly put the list of steps on mac book pro? Using option key instead of the Alt does not seem to work

**Comment by bsnyder** [comment]
Same here -- option/alt key on macOS does not work.

**Comment by CameosVatican** [comment]
Agreed - _still_ does not work on Mac September 2022 - come on, how about it, James?!!

**Comment by sun** [comment]
Works fine for me on macOS.
